log/keyvalue_bench_test.go

```diff
@@ -232,3 +233,65 @@ func BenchmarkValueEqual(b *testing.B) {
 		}
 	}
 }
+
+func BenchmarkKeyValueFromAttribute(b *testing.B) {
+	testCases := []struct {
+		desc string
+		kv   attribute.KeyValue
+		want log.KeyValue
```

Comment:
This doesn't seem used. Is there a reason to keep it here?

Reply:
https://github.com/open-telemetry/opentelemetry-go/pull/6180/commits/904113d6a10575108cfa3c813c25ea6b3b38b63b